### ISSUE TITLE
GPU renderer causes PC to black screen, freeze, and flicker when multi boxing/logging

### ISSUE THREAD

**Issue by ANRH** [runelite/runelite#6551]
The bug freezes, black screens, and flickers my screen for several seconds when I have two Runelite clients open (both of them having GPU enabled) and drag any other application.

**Steps to reproduce the behavior:**
1. Enable the GPU plugin on two clients.
2. Open a different application and drag the window around.

NVIDIA GeForce GTX 1060
Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz

**Comment by deathbeam** [comment]
Can you retest this with <https://github.com/runelite/runelite/commit/3286594d6b583aa261483a73ddf30823c102e923> @ANRH ?

**Comment by ANRH** [comment]
I may be wrong, but it doesn't look like anything changes with that, @deathbeam

**Comment by ANRH** [comment]
No change, @deathbeam.

**Comment by deathbeam** [comment]
Did you actually tested master branch and built the client?

**Comment by ANRH** [comment]
![screenshot_204](https://user-images.githubusercontent.com/45132385/48737887-ad1e5400-ec68-11e8-9a5c-5a886e5e24bd.png)

Does this look right?

**Comment by deathbeam** [comment]
Yeah @ANRH

**Comment by ANRH** [comment]
Then the bug is still there, @deathbeam

**Comment by deathbeam** [comment]
Alright, thanks for testing it

**Comment by MESLewis** [comment]
Does this happen on the latest release of RuneLite?

**Comment by Extarys** [comment]
Not sure if this is the exact cause and same isssue as I'm still investigating but I can't think of anything else that could cause this.
Ever since I started using this I get black screen, only when this is opened. Sometimes it happens after 2-3 hours of game play.

I thought it was caused by the latest march windows 10 update that cause BSOD but I'm getting a black screen, not blue screen.
I'll update if I find anything more - to be clear *I'm not 100% sure RL GPU is the cause*
